src/Components/DnDLayout/GridLayout.tsx

```diff
@@ -100,6 +101,12 @@ const GridLayout = ({ isLayoutLocked = false, layoutType = 'landingPage' }: { is
   const addNotification = useAddNotification();
   const setCurrentlyUsedWidgets = useSetAtom(currentlyUsedWidgetsAtom);
   const { analytics } = useChrome();
+  const isITLess = useFlag('platform.widget-layout.itless');
```

Comment:
Hmmm
I wonder if we could pass this string as a prop from the fed-modules config as we are doing for learning resources.

Its much cleaner and we do not need a feature flag on frontend all.

Reply:
That sounds great to me. I'd rather avoid having to use featureFlags if I can avoid it. It makes sense to load this from the frontend deploy file